puppet/modules/kubernetes/templates/kubelet.service.erb

```diff
@@ -136,24 +140,25 @@ ExecStart=<%= scope['kubernetes::_dest_dir'] %>/kubelet \
   --eviction-soft-grace-period=<%= @eviction_soft_grace_period.join(',') %> \
   --eviction-max-pod-grace-period=<%= @eviction_max_pod_grace_period %> \
   --eviction-pressure-transition-period=<%= @eviction_pressure_transition_period %> \
-<%        end -%>
-<%    end -%>
+<% end -%>
+<% end -%>
 <% end -%>
 <%
-  # build minumum reclaim command line
-  @eviction_minimum_reclaim = []
+    # build minumum reclaim command line
+    @eviction_minimum_reclaim = []
 
-  @eviction_minimum_reclaim << "memory.available=#{@eviction_minimum_reclaim_memory_available}" unless @eviction_minimum_reclaim_memory_available.nil? or @eviction_minimum_reclaim_memory_available == 'nil'
-  @eviction_minimum_reclaim << "nodefs.available=#{@eviction_minimum_reclaim_nodefs_available}" unless @eviction_minimum_reclaim_nodefs_available.nil? or @eviction_minimum_reclaim_nodefs_available == 'nil'
-  @eviction_minimum_reclaim << "nodefs.inodesFree=#{@eviction_minimum_reclaim_nodefs_inodes_free}" unless @eviction_minimum_reclaim_nodefs_inodes_free.nil? or @eviction_minimum_reclaim_nodefs_inodes_free == 'nil'
+    @eviction_minimum_reclaim << "memory.available=#{@eviction_minimum_reclaim_memory_available}" unless @eviction_minimum_reclaim_memory_available.nil? or @eviction_minimum_reclaim_memory_available == 'nil'
+    @eviction_minimum_reclaim << "nodefs.available=#{@eviction_minimum_reclaim_nodefs_available}" unless @eviction_minimum_reclaim_nodefs_available.nil? or @eviction_minimum_reclaim_nodefs_available == 'nil'
+    @eviction_minimum_reclaim << "nodefs.inodesFree=#{@eviction_minimum_reclaim_nodefs_inodes_free}" unless @eviction_minimum_reclaim_nodefs_inodes_free.nil? or @eviction_minimum_reclaim_nodefs_inodes_free == 'nil'
 -%>
 <% if @eviction_minimum_reclaim.length > 0 -%>
   "--eviction-minimum-reclaim=<%= @eviction_minimum_reclaim.join(',') %>" \
 <% end -%>
+
 <% if @_feature_gates != [] -%>
-  --feature-gates=<%= @_feature_gates.join(',') %> \
+  --feature-gates=<%= @_feature_gates.join(',') %>
+<% end -%>
 <% end -%>
-  --logtostderr=true
```

Comment:
We always need to end with no back slash. Can you make this flag the last parameter again